packages/pluggableWidgets/heatmap-chart-web/cypress/support/commands.js

```diff
@@ -0,0 +1,25 @@
+// ***********************************************
```

Comment:
Do we need this file? 😄

Reply:
Ouch! Good one! It's not necessary, I'll remove it.